collector/nvidia.go

```diff
@@ -0,0 +1,305 @@
+// +build windows
+
+package collector
+
+/*
+#include "nvml.h"
+#include <windows.h>
+
+// definition of required import symbols
```

Comment:
We have a pattern of putting Cgo-bits in the `headers/` directory, and expose a "nice" Go interface from there, so the collector logic gets easier to read

Reply:
ok, i can rework that.

Reply:
So, at this point the repo does not have a `headers/` directory. Thus I have no idea what exactly should go in there. Can you give me a link to any other project/repo that has such a structure for an example? Sorry, I'm primarily a C++ dev and really do not know the guidelines for Go/C interop.

Reply:
Hm, perhaps you need to rebase? Looks like this branch started off in February and it might not have existed then, but it does now. There's some background in [this PR](https://github.com/prometheus-community/windows_exporter/pull/702#issuecomment-769785337)